plugins/inventory/iocage.py

```diff
@@ -233,6 +255,7 @@ def get_inventory(self, path):
         if get_properties:
             for hostname, host_vars in results['_meta']['hostvars'].items():
                 cmd_get_properties = cmd.copy()
+                cmd_get_properties.append(self.IOCAGE)
```

Comment:
So `sudo` is not needed here?

Reply:
No. I see no restrictions in commands *iocage get ...*. However, I haven't reviewed the *iocage* code.

Reply:
I was thinking more of that accessing the iocage binary might need `sudo` rights. But if that's unlikely then that's fine :)

Reply:
IP4 is the only attribute that depends on a running jail. There are no more *jexec* calls except the one mentioned above when the list of jails is created.